editor/src/messages/preferences/preferences_message_handler.rs

```diff
@@ -14,6 +14,7 @@ pub struct PreferencesMessageHandler {
 	pub use_vello: bool,
 	pub vector_meshes: bool,
 	pub graph_wire_style: GraphWireStyle,
+	pub zoom_wheel_rate: Option<f64>,
```

Comment:
Why is this an option? Is there a case when we'd have a zoom wheel rate of None?